Rubberduck.VBEditor.VB6/SafeComWrappers/VB/AddIn.cs

```diff
@@ -9,8 +12,18 @@ public class AddIn : SafeComWrapper<VB.AddIn>, IAddIn
         public AddIn(VB.AddIn target, bool rewrapping = false) 
             : base(target, rewrapping)
         {
+            CommandBarLocations = new ReadOnlyDictionary<CommandBarSite, CommandBarLocation>(new Dictionary<CommandBarSite, CommandBarLocation>
+            {
+                {CommandBarSite.MenuBar, new CommandBarLocation(1, 30009)},
```

Comment:
Maybe use named arguments here. Although they are redundant they would better convey what these numbers stand for.

Reply:
Will do